airbyte-webapp/src/hooks/useSegment.tsx

```diff
@@ -1,3 +1,5 @@
+/// <reference types="@types/segment-analytics" />
```

Comment:
:information_source: Without `types` in the `tsconfig.json` (which was the case beforehand) all `@types/...` packages get automatically imported. That's not the case anymore now, so we'll reference the only missing one in this file explicitally.